v2/thirdparty/custom-ui/multi-tenancy.mdx

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
We should change the "Social / SSO login" page in the custom-ui section to be it's own section which contains the following pages:
- Social login (ex: Doogle, Apple, Facebook)
- Multi tenant SSO (ex: Okta, Active directory)
-